gateway/api/middleware.go

```diff
@@ -51,70 +96,188 @@ func (a *Api) Authenticate(c *gin.Context) {
 		)
 		c.Set(pgusers.ContextLoggerKey, zaplogger.Sugar())
 	}
-	switch openapi.RoleType(roleName) {
-	case openapi.RoleStandardType: // noop
-	case openapi.RoleUnregisteredType:
-		if !ctx.IsEmpty() {
-			break
+
+	c.Set(storagev2.ContextKey,
+		storagev2.NewContext(ctx.UserSubject, ctx.OrgID).
+			WithUserInfo(ctx.UserName, ctx.UserEmail, string(ctx.UserStatus), ctx.UserPicture, ctx.UserGroups).
+			WithSlackID(ctx.UserSlackID).
+			WithOrgName(ctx.OrgName).
+			WithOrgLicense(ctx.OrgLicense).
+			// WithApiURL(a.IDProvider.ApiURL).
```

Comment:
good catch. Nevertheless, even though this data is the same loaded in API_URL, I disagree it would come from `IDProvider`. Maybe it should come directly from the config with the configured API_URL.